libdeno/binding.cc

```diff
@@ -105,10 +105,13 @@ std::string EncodeExceptionAsJSON(v8::Local<v8::Context> context,
       CHECK(frame_obj
                 ->Set(context, v8_str("functionName"), frame->GetFunctionName())
                 .FromJust());
-      CHECK(frame_obj
-                ->Set(context, v8_str("scriptName"),
-                      frame->GetScriptNameOrSourceURL())
-                .FromJust());
+      // scriptName can be empty in special conditions e.g. eval
+      auto scriptName = frame->GetScriptNameOrSourceURL();
+      if (scriptName.IsEmpty()) {
+        scriptName = v8_str("<unknown>");
+      }
+      CHECK(
+          frame_obj->Set(context, v8_str("scriptName"), scriptName).FromJust());
```

Comment:
Can you please add something like this to libdeno_test.cc
```c
TEST(LibDenoTest, EncodeErrorBug) {
  Deno* d = deno_new(deno_config{0, empty, nullptr});
  EXPECT_FALSE(deno_execute(d, d, "a.js", "eval('a')"));
  EXPECT_STREQ(deno_last_exception(d),
               "{\"message\":\"Error: boo\","
               "\"frames\":[{\"line\":3,\"column\":7,"
               "\"functionName\":\"\",\"scriptName\":\"a.js\","
               "\"isEval\":false,"
               "\"isConstructor\":false,\"isWasm\":false}]}");
  deno_delete(d);
}
```

Reply:
Was working on an output diff test, but I guess a libdeno test sounds more reasonable...

Reply:
All checks finally turn green.